src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
```

Comment:
you can remove box-sizing: border-box; 
and use beter this approach + add this code to reset.scss

```
html {
  box-sizing: border-box;
}

*,
*::after,
*::before {
  box-sizing: inherit;
}
```

Reply:
thanks